### ISSUE TITLE
using api layer only without ui

### ISSUE THREAD

**Issue by pinale** [KorzhCom/EasyData#75]
Is it possible to use only the automatically generated endpoints without using the UI layer.
Id like to call these crud enpoints from my angular client.
If yes, can you kindly provide an example please.
Thanks for your great library

**Comment by korzh** [comment]
Yes, it's possible to use API separately but we don't have any documentation or any example yet. 

Currently, you can figure it out by observing (with the Developer Tools panel in your browser) the requests sent from the client-side to the backend in any of our sample projects.

I will let you know when we publish the docs or create a sample that uses API without the JavaScript UI part.

**Comment by pinale** [comment]
It could be usefull to have a swagger file along with the autogenerated endpoints, 
in order to permit everyone to create client code with some automation tools.
It could be a great improvement in my opinion
